cider-velox/src/TypeConversions.h

```diff
@@ -56,7 +56,9 @@ inline const char* getArrowFormat(::substrait::Type& typeName) {
     case ::substrait::Type::KindCase::kTimestamp:
       return "ttu";  // MICROSECOND
     default:
-      throw std::runtime_error("Conversion is not supported yet");
+      throw std::runtime_error(
```

Comment:
please use velox macro to throw exception

Reply:
fix.